### PR TITLE
Fix for svgo returned promise

### PR DESCRIPTION
**Pull request by strarsis**
This PR fixes the usage of svgo which returns a Promise instead 
supporting a callback function as 2nd argument.

Edit: Because some may use the old and some the recent/new versions of svgo, 
both cases, callback + promise need to be handled intelligently. - This is still missing in this PR...

### CODE REVIEW

**Comment by kevva** [comment]
They haven't released the Promise based API yet.

**Comment by strarsis** [comment]
@kevva: svgo version 1.0.0 has just been released.

**Comment by kevva** [comment]
@strarsis, nice! Yes, although I think you have to do some changes.

**Comment by strarsis** [comment]
@kevva: Yes, the code must take older versions of svgo into account 
that don't return Promises but expect a callback instead.

File: index.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ module.exports = opts => buf => {
 	const svgo = new SVGO(opts);
 
 	return new Promise((resolve, reject) => {
```
Comment (kevva) [inline index.js]:
```js
return svgo.optimize(buf).then(res => Buffer.from(res.data));
```

**Comment by kevva** [comment]
@strarsis, we don't care about that. Just bump `svgo` in `package.json` to the newest version.

**Comment by strarsis** [comment]
@kevva: Added the change, bumped svgo version and fixed conflicts.

**Comment by strarsis** [comment]
@kevva: Ready to merge.

**Comment by kevva** [comment]
Thank you very much @strarsis 🎉

**Comment by strarsis** [comment]
@kevva: Please-release 👍

**Comment by kevva** [comment]
@strarsis, done!

**Pull request merged**